source/javascripts/controllers/_MainController.js.erb

```diff
@@ -106,6 +106,7 @@ import datadogRumCustomTiming from "../utils/datadogCustomRumTiming.ts";
 			viewModel.currentUser;
 			viewModel.productTourShown;
 			viewModel.isGuidedOnboardingEnabled = false;
+			viewModel.productTourDismissed = false;
```

Comment:
We should be consistent in our naming. This one's fine, but I think `isGuidedOnboardingEnabled` should be `guidedOnboardingEnabled`, and I guess I missed that PR! :)

Reply:
updated